models/issue_assignees.go

```diff
@@ -162,6 +162,10 @@ func (issue *Issue) changeAssignee(sess *xorm.Session, doer *User, assigneeID in
 	mode, _ := accessLevel(sess, doer.ID, issue.Repo)
 	if issue.IsPull {
 		if err = issue.loadPullRequest(sess); err != nil {
+			// If pull request is not yet created - don't fail here, skip webhooks
+			if IsErrPullRequestNotExist(err) {
```

Comment:
If it's creation not modification and `if IsErrPullRequestNotExist(err)` then return nil, other will return err

Reply:
Added fourth argument to function which determines whenever assignee is added in the middle of creation or not. Also this made ```if IsErrPullRequestNotExist(err)``` check not necessary - no reason to even try to load pull request if it's in creation.